app/infrastructure/qna/qna_generator.py

```
@@ -0,0 +1,460 @@
+    raw_prompts: List[str] = []
+    raw_responses: List[str] = []
+    
+    merged: Dict[str, QAPair] = {}
+    
+    for i, chunk_qs in enumerate(_chunk(questions, batch_size), start=1):
+        try:
+            chunk_pairs, prompt, audit = _ask_batch(
+                together_client, model, analysis_text, chunk_qs,
+                temperature=temperature, max_tokens=max_tokens
+            )
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)